front-end/src/components/Pricing/PricingCard/PricingCard.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { nanoid } from 'nanoid';
+
+//Services
+import { sendPaymentData } from 'services/paymentService';
+
+//Styles
+import styles from './PricingCard.module.scss';
+import Button from '@mui/material/Button';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
+
+const PricingCard = ({ pricing, user, teacher }) => {
+  const { name, price, currency, description } = pricing;
+
+  const [payment, setPayment] = useState({
+    data: '',
+    signature: '',
+  });
+
+  const dae = JSON.stringify({
```

Comment:
What it is dae?

Reply:
This is name of the liqpayfield with additional information of the payment

Reply:
Changed dae to userInfo